src/Unicode/Utf8EncodingScheme.dfy

```diff
@@ -0,0 +1,67 @@
+// RUN: %dafny /compile:0 "%s"
+
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT
+*******************************************************************************/
+
+include "../Collections/Sequences/Seq.dfy"
+include "../BoundedInts.dfy"
+
+include "Unicode.dfy"
+include "Utf8EncodingForm.dfy"
+
+/**
+ * The Unicode encoding scheme that serializes a UTF-8 code unit sequence in exactly the same order as the code unit
+ * sequence itself.
+ *
+ * Because the UTF-8 encoding form already deals in ordered byte sequences, the UTF-8 encoding scheme is trivial. The
+ * byte ordering is already obvious and completely defined by the UTF-8 code unit sequence itself. The UTF-8
+ * encoding scheme is defined merely for completeness of the Unicode character encoding model.
```

Comment:
And also because `unit8` and `bv8` are not the same type? :)

Reply:
Noted in latest commits.